src/views/Bitcoin.js

```diff
@@ -0,0 +1,154 @@
+import React, { useState } from 'react';
+import { Grid, Text } from 'indigo-react';
+import { useWallet } from 'store/wallet';
+import { usePointCursor } from 'store/pointCursor';
+
+import { useLocalRouter } from 'lib/LocalRouter';
+
+import ViewHeader from 'components/ViewHeader';
+import View from 'components/View';
+import WarningBox from 'components/WarningBox';
+
+import CopiableAddressWrap from 'components/CopiableAddressWrap';
+import Highlighted from 'components/Highlighted';
+
+import { PsbtInput } from 'form/Inputs';
+import BridgeForm from 'form/BridgeForm';
+
+import * as need from 'lib/need';
+import * as bitcoin from 'bitcoinjs-lib';
+import * as bs58check from 'bs58check';
+import ob from 'urbit-ob';
+import { Buffer } from 'buffer';
+import { composeValidator, buildPsbtValidator } from 'form/validators';
+
+function xpubToYpub(xpub) {
+  var data = bs58check.decode(xpub);
+  data = data.slice(4);
+  data = Buffer.concat([Buffer.from('04b24746', 'hex'), data]);
+  return bs58check.encode(data);
+}
+
+export default function Bitcoin() {
+  const { pop } = useLocalRouter();
+  const { urbitWallet } = useWallet();
+  const { pointCursor } = usePointCursor();
+
+
+  const empty = new bitcoin.Psbt();
+
+  const [signedTx, setSignedTx] = useState('');
+  const [psbt, setPsbt] = useState(empty);
+
+  const pointName = ob.patp(need.point(pointCursor));
+
+  const privKey = need.urbitWallet(urbitWallet).bitcoin.keys.private;
+  const pubKey = need.urbitWallet(urbitWallet).bitcoin.keys.public;
+  const chain = need.urbitWallet(urbitWallet).bitcoin.keys.chain;
+
+  const hd = bitcoin.bip32.fromPrivateKey(
+    Buffer.from(privKey, 'hex'),
+    Buffer.from(chain, 'hex'),
+    bitcoin.networks.bitcoin
+  );
+
+  const zPub = xpubToYpub(
```

Comment:
So is it a ypub or a zpub?

My understanding of these is that they're just xpubs but for different bitcoin wallet/signing(?) protocols. Are they interchangeable like this, for our use case?

Reply:
Yeah it's a zpub, I'm just dumb, fixed.

They're kinda interchangeable, but we're doing BIP84 so the correct one to use is zpub.